templates/update.html

```diff
@@ -25,9 +25,12 @@ <h1>Update {{ mod.name }}</h1>
         <div class="row">
             <div class="col-md-4">
                 <h2 class="control-label">Mod version</h2>
+                <p>Previous: {{ mod.default_version.friendly_version }}</p>
```

Comment:
I think the Previous tip is more likely to help if it's closer to the input box. Can we move it to the right? Yes it would collide with the warning then, but the warning won't show up most of the time.

Reply:
So below the text box where the warning is currently, and make the warning pop up just next of it?

I'm also trying to find a way to make the warning show without shifting the layout of the entire page.
Moving the "Previous" label next to it would help with that, but the button still causes it to shift.
Setting a `margin` to match the button is a bit fragile.
Ideas?

Reply:
Looks fine now, and I think it's OK to annoy the user in this instance. :grinning: